main.cpp

```diff
@@ -510,29 +635,36 @@ Thing 10) truck
  */
 struct Truck
 {
-    //wheels
+    
     int wheels = 4;
-    //engine
     int engine = 1;
-    //seats
     int seats = 4;
-    //steering wheel
     int steeringWhee = 1;
-    //fuel system
     int fuelSystem = 1;
 
-    //carry people
+    
     void maxOccupancy(int = 4);
-    //tow vehicles
     void towVehcles(int maxPower = 100);
-    //drive long distance
-    double driveLongDistance();
+    void driveLongDistance(double distance);
 
 };
 
+void Truck::maxOccupancy(int)
+{
+    int potentialOccupy = 100;
+        std::cout << potentialOccupy << std::endl;
+}
 
+void Truck::towVehcles(int maxPower)
+{
+    if (maxPower > 100)
+        maxPower = 200;
+}
 
-
+void Truck::driveLongDistance(double distance)
+{
+    distance = 10;
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise